src/vaev-engine/style/selector-impl.cpp

```diff
@@ -0,0 +1,26 @@
+module Vaev.Engine;
+
+import Karm.Core;
+import :style.selector;
+
+namespace Vaev::Style {
+
+bool Pseudo::operator==(Pseudo const&) const = default;
+
+bool Infix::operator==(Infix const& other) const = default;
+
+bool Nfix::operator==(Nfix const& other) const{
+    // FIXME: compile error
```

Comment:
@sleepy-monax I'll need ur eyes here

Reply:
before merging, after the final review, ill switch to defaulting. its quicker than trying to set up the correct version